apps/publisher/themes/default/js/copy-asset.js

```diff
@@ -19,13 +19,11 @@ $(document).ready(function() {
                 }),
                 type: 'POST',
                 success: function(response) {
-                    messages.alertSuccess('Asset version created successfully!');
+                    messages.alertSuccess('Asset version created successfully!,You will be redirected to new asset details page in few seconds.....');
```

Comment:
Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/780298fc7ac427dd7facbe4c41cf6782f7ce5f44)